core/arch/arm/dts/stm32mp157a-dk1.dts

```diff
@@ -248,3 +248,32 @@
 	pinctrl-0 = <&uart4_pins_a>;
 	status = "okay";
 };
+
+&bsec {
+	mac_addr: mac_addr@e4 {
+		reg = <0xe4 0x6>;
+		status = "okay";
+		secure-status = "okay";
+	};
+	/* Spare field to align on 32-bit OTP granularity  */
+	spare_ns_ea: spare_ns_ea@ea {
+		reg = <0xea 0x2>;
+		status = "okay";
+		secure-status = "okay";
+	};
```

Comment:
Why is this node needed? Each node has its reg value with address and size, it's not like a C struct where you have to reserve some space. So I would think things would work the same if you remove `spare_ns_ea`.

Reply:
Ok, a bit unclear. I'm sending a fixup that should clarify.

Note: i fixup change i'll moce the `mac_addr` node frmo board DTS to SoC DTSI, since it is defined in SoC manual and is not board specific.